src/Knp/Rad/ResourceResolver/EventListener/ResourcesListener.php

```diff
@@ -105,4 +112,12 @@ protected function castParameter($parameter)
             }
         }
     }
+
+    protected function isNullValue($resource)
+    {
+        return null === $resource ||
+            (is_array($resource) && empty($resource)) ||
+            ($resource instanceof \Countable && 0 === count($resource))
```

Comment:
Cf. my answer above. And I wanted to avoid `empty` on something else than an array cause I don't think `false` as value or `0` should throw an exception.